airbyte-integrations/connectors/destination-s3-glue/src/main/java/io/airbyte/integrations/destination/s3_glue/MetastoreOperations.java

```diff
@@ -1,13 +1,15 @@
-package io.airbyte.integrations.destination.s3;
+package io.airbyte.integrations.destination.s3_glue;
 
 import com.fasterxml.jackson.databind.JsonNode;
 
+// TODO (itaseskii) allow config based implementation of different metastores i.e Hive, Nessie, etc.
```

Comment:
If the intent is to support different metastores then it might make sense to change the name from `s3_glue` to `s3_lakehouse` or `s3_metastore` so that there isn't the hardcoded assumption that it only works with AWS Glue.

Reply:
I fully agree with this and I was thinking the same thing but I would prefer if we rename it once we actually have another metastore implementation since that is not for certain...

Reply:
Makes sense :+1: